third_party/bigframes_vendored/pandas/core/generic.py

```diff
@@ -844,8 +856,9 @@ def isna(self) -> NDFrame:
             dtype: boolean
 
         Returns:
-            Mask of bool values for each element that indicates whether an
-            element is an NA value.
+            NDFrame:
```

Comment:
I think this should actually be bigframes.pandas.DataFrame or bigframes.pandas.Series. I don't think we have any code that returns this superclass directly.

```suggestion
            bigframes.pandas.DataFrame or bigframes.pandas.Series:
```